jax/_src/numpy/fft.py

```diff
@@ -400,15 +400,133 @@ def ifft(a: ArrayLike, n: int | None = None,
   return _fft_core_1d('ifft', xla_client.FftType.IFFT, a, n=n, axis=axis,
                       norm=norm)
 
-@implements(np.fft.rfft)
+
 def rfft(a: ArrayLike, n: int | None = None,
          axis: int = -1, norm: str | None = None) -> Array:
+  r"""Compute a one-dimensional discrete Fourier transform for real input.
+
+  JAX implementation of :func:`numpy.fft.rfft`.
+
+  Args:
+    a: real-valued input array.
+    n: int. Specifies the dimension of the result along ``axis``. The dimension
+      of the array along ``axis`` is ``(n/2)+1``, if ``n`` is even and ``(n+1)/2``,
+      if ``n`` is odd. If not specified, ``n`` defaults to the dimension of ``a``
+      along ``axis``.
```

Comment:
It might be easier to think about this as the number of inputs that are used, rather than related to the dimension of the output. Right now, I think it's a bit confusing to say that `n` "specifies" the output dimension, when the output dimension is a function of `n`.

Reply:
Is it more accurate to say that `n` specifies the effective input size?

Reply:
Thanks! Changed the description of `n` as `n: int. Specifies the effective dimension of the input along ``axis``. If not specified, it will default to the dimension of input along ``axis``.` and the discussion of output size is moved to "Returns:".